setup.py

```diff
@@ -166,7 +166,6 @@ def generate_build_ext(pkg_dir: Path, pkg_name: str):
     else:
         # flags for Linux and Mac
         cflags += [
-            "-std=c++20",
```

Comment:
well it is added below. the temp fix is that because the musllinux image uses an old version of GCC (9.x) that does not yet support `-std=c++20` but only `-std=c++2a` (`-std=c++20` is supported from GCC 10 onwards). as soon as there's a new musllinux image out with a more recent GCC version, we can switch back to `-std=c++20`. until then, only a subset of C++20 features are supported, unfortunately